### PR TITLE
chore: create deployment dockerfiles for preview builds

### PR DESCRIPTION
**Pull request by wreality**
This makes changes to the docker configuration (note, this does not affect Lando at all) to support creating preview builds.  Documentation for preview builds is also added in this PR: 

https://deploy-preview-152--eloquent-galileo-3be94a.netlify.app/contributing/#preview-environments

### CODE REVIEW

**Comment by wreality** [comment]
@gmeben @Sruthin86 @amareshjoshi 

This is ready for review if anyone wants to take a look.  Really as far as CCR goes, it's some minor changes to config and additional documentation.  But, make sure the docs make sense, play with the feature if you like (note, it'll only work on this PR until this PR is merged), and let me know if something doesn't seem like it's working quite right.

https://deploy-preview-152--eloquent-galileo-3be94a.netlify.app/contributing/#preview-environments

**Review by gmeben — APPROVED** [review]
We'll need to look into the markdown linting discrepancies.

File: docs/contributing/README.md [hunk 1]
```diff
@@ -86,6 +86,41 @@ The MailHog interface will be available at <http://mailhog.ccr.lndo.site/> once
...
+If the pull request does not meet the above rules, `CCR-Droid` will comment on the pull request and remove the `pr-preview` label.  If the pull request meets the criteria, `CCR-Droid` will request the deployment for you.
+
+![pending deployment screenshot](./images/preview_in_progress.jpg)
+
+After `CCR-Droid` creates the deployment, a link will appear above the checks section.
+
+![deployment ready screenshot](./images/preview_ready.jpg)
+
+#### Mail
+Mailhog captures mail from preview deployments to prevent abuse.  Visit <https://mail.gh.ccrproject.dev> to access the Mailhog server and view any email sent by preview deployments.
```
[the remark this answers is not in the document]
Comment (wreality) [reply docs/contributing/README.md:112]: Try installing the [markdownlint vscode extension](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint).  That should make vscode respect the existing rules, but then lets totally open a PR request for the rules changes, I've no objections to them, especially if there are rules for them and my editor will then just fix them for me. :-)

[the remark this answers is not in the document]
Comment (gmeben) [reply docs/contributing/README.md:112]: Yes, I installed that and it's not respecting the existing rules.

**Pull request merged**